packages/cli/tests/commonlib/keyVaultValidator.ts

```diff
@@ -115,27 +115,30 @@ export class KeyVaultValidator {
     const identityTokenCredential = await tokenProvider.getIdentityCredentialAsync();
     const tokenToGetSecret = (await identityTokenCredential?.getToken(keyvaultScope))?.token;
 
-    const m365ClientSecretName =
-      (await getProvisionParameterValueByKey(
-        this.projectPath,
-        this.env,
-        provisionParametersKey.m365ClientSecretName
-      )) ?? "m365ClientSecret";
-    const keyVaultSecretResponse = await this.getKeyVaultSecrets(
-      this.keyVault.vaultUri,
-      m365ClientSecretName,
-      tokenToGetSecret as string
-    );
-    chai.assert.exists(keyVaultSecretResponse);
-    const expectedM365ClientSecret = await CliHelper.getUserSettings(
-      `${PluginId.Aad}.${StateConfigKey.clientSecret}`,
-      this.projectPath,
-      this.env
-    );
-    chai.assert.equal(keyVaultSecretResponse, expectedM365ClientSecret);
-
     const activeResourcePlugins = await getActivePluginsFromProjectSetting(this.projectPath);
```

Comment:
Please check activeResourcePlugins once enter this function and once not includes both AAD and Bot, return directly without assigning "get secret" permission. You can check [this change](https://github.com/OfficeDev/TeamsFx/pull/5024/files).

Reply:
Fixed, thx！